backend/src/interface/admin_notification/mod.rs

```diff
@@ -1 +1,33 @@
 //! This interface allows administrators to be notified of reporting requests.
+
+use async_trait::async_trait;
+use uuid::Uuid;
+
+use crate::util::ReportReason;
+
+/// Interface for notification of administrators.
+#[async_trait]
+pub trait AdminNotification {
+    /// Notifies an administrator about a newly reported image and the response automatically taken.
+    async fn notify_admin_image_report(info: ImageReportInfo);
```

Comment:
Async not necessary for functions without return?(!)

Reply:
Functions without an explicit return type return `()`. You cannot `await` an async function in a synchronous one.